src/components/scenes/FioAddressDetailsScene.js

```diff
@@ -86,10 +104,13 @@ class FioAddressDetails extends React.Component<Props, LocalState> {
 
   render() {
     const { theme, route } = this.props
-    const { fioAddressName } = route.params
+    const { fioAddressName, bundles } = route.params
+    const styles = getStyles(theme)
+    const bundlesLabel = `${s.strings.fio_address_details_screen_bundles}: ${bundles}`
```

Comment:
Please change all variables that have `bundles` to `bundledTxs` or `bundled_txs` bundles is not a proper name